db/index.js

```diff
@@ -137,6 +137,75 @@ async function getUserByUsername(username) {
   }
 }
 
+//order by id
+async function getOrderById(id) {
```

Comment:
These 60 lines of code that I wrote is showing up as new in this pull request of yours but I pushed this up in the last dev merge so I am wondering why it's showing up as new? Will this create any issues?